assets/css/tablesorter.theme.glotpress.css

```diff
@@ -19,6 +19,7 @@ GlotPress Theme
 	cursor: pointer;
 	text-align: center;
 	border: 1px solid #eee;
+	padding-right: 20px !important;
```

Comment:
The use of an `!important` should be avoided. We're overwriting the padding behaviour [here](https://github.com/GlotPress/GlotPress-WP/blob/2.2.0/assets/css/style.css#L214) so we have to restore the original right padding:
```css
table.translation-sets.tablesorter-glotpress thead th {
    padding-right: 20px;
}
```